packages/devtools_app/lib/src/code_size/code_size_screen.dart

```diff
@@ -308,97 +189,201 @@ class CodeSizeBodyState extends State<CodeSizeBody>
     );
   }
 
-  Widget _buildTreemap({@required bool showDiff}) {
+  Widget _buildTreemap() {
     return LayoutBuilder(
       key: treemapKey,
       builder: (context, constraints) {
         return Treemap.fromRoot(
-          rootNode: showDiff ? diffRoot : snapshotRoot,
+          rootNode: snapshotRoot,
           levelsVisible: 2,
           isOutermostLevel: true,
           height: constraints.maxHeight,
-          onRootChangedCallback: showDiff
-              ? controller.changeDiffRoot
-              : controller.changeSnapshotRoot,
+          onRootChangedCallback: controller.changeSnapshotRoot,
         );
       },
     );
   }
+
+  Widget _buildImportSnapshotView() {
+    return Padding(
+      padding: const EdgeInsets.all(defaultSpacing),
+      child: Column(
+        children: [
+          Flexible(
+            child: FileImportContainer(
+              title: 'Snapshot',
+              actionText: 'Analyze Snapshot',
+              onAction: controller.loadFakeTree,
+            ),
+          ),
+          const SizedBox(height: defaultSpacing),
+          _buildHelpText(),
+        ],
+      ),
+    );
+  }
+
+  Column _buildHelpText() {
+    return Column(
+      children: [
+        Text(
+          'We currently only support instruction sizes and v8 snapshot profile outputs.',
+          style:
+              TextStyle(color: Theme.of(context).colorScheme.chartAccentColor),
+        ),
+      ],
+    );
+  }
+}
+
+class DiffView extends StatefulWidget {
+  @override
+  _DiffViewState createState() => _DiffViewState();
 }
 
-class FileImportContainer extends StatelessWidget {
-  const FileImportContainer({
-    @required this.title,
-    @required this.onImport,
-    this.importedFile,
-    Key key,
-  }) : super(key: key);
+class _DiffViewState extends State<DiffView> with AutoDisposeMixin {
+  @visibleForTesting
+  static const treemapKey = Key('Code Size Treemap');
 
-  final String title;
+  static const initialFractionForTreemap = 0.67;
+  static const initialFractionForTreeTable = 0.33;
```

Comment:
these consts are reused. Just put them as top level constants at the top of this file